webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/openmrsSearch.js

```diff
@@ -855,17 +907,18 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
 
         _doKeyDown: function() {
             //the user is using the mouse and they also want to use up/down keys?, dont support this
-            if(this.hoverRowSelection != null)
+            if(this.hoverRowSelection != null){
                 return;
+			}
 
             var prevRow = this.curRowSelection;
             //if we are on the last page, and the last row is highlighted, do nothing
             if(this._getCurrVisiblePage() == this._table.numberOfPages && prevRow >= (this._results.length-1) && this._results.length > 1)
-                return;
+			{ return;}
```

Comment:
You could return null instead of {return}

Reply:
Ok thanks